remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
```

Run commands, upgrade to a complete sentence:

```suggestion
                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` by running `ddev config global --simple-formatting`."
```